plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,234 @@
+    required: false
+    type: int
+  backup_jobs:
+    description:
+      - The backup_jobs value is true or false.
+      - If V(true), the module will return all backup jobs information.
+      - If V(false), the module will parse all backup jobs based on VM IDs and return a list of VMs' backup information.
+    required: false
+    default: false
+    type: bool
+
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+  - community.general.proxmox.actiongroup_proxmox
+"""
+
+EXAMPLES = """
+- name: Print all backup information by VM id and VM name
+  proxmox_backup_info:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+
+- name: Print proxmox backup information for a specific VM based on its name
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      vm_name: 'mailsrv'
+
+- name: Print proxmox backup information for a specific VM based on its VM id
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      vm_id: '150'
+
+- name: Print proxmox all backup job information
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      backup_jobs: true
+"""
+
+RETURN = """
+---
+backup_info:
+  description: The return value provides backup job information based on VM ID or VM name, or total backup job information.
+  returned: always, but can be empty
+  type: list
+  elements: dict
+  contains:
+    bktype:
+      description: The type of the backup.
+      returned: on success
+      type: str
+    enabled:
+      description: 1 if backup is enabled else 0.
+      returned: on success
+      type: int
+    id:
+      description: The backup job id.
+      returned: on success
+      type: str
+    mode:
+      description: The backup job mode such as snapshot.
+      returned: on success
+      type: str
+    next-run:
+      description: The next backup time.
+      returned: on success
+      type: str
+    schedule:
+      description: The backup job schedule.
+      returned: on success
+      type: str
+    storage:
+      description: The backup storage location.
+      returned: on success
+      type: str
+    vm_name:
+      description: The VM name.
+      returned: on success
+      type: str
+    vmid:
+      description: The VM vmid.
+      returned: on success
+      type: int
+"""
+
+from datetime import datetime
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible_collections.community.general.plugins.module_utils.proxmox import (
+    proxmox_auth_argument_spec, ProxmoxAnsible, HAS_PROXMOXER, PROXMOXER_IMP_ERR)
+
+
+class ProxmoxBackupInfoAnsible(ProxmoxAnsible):
+
+    # Get all backup information
+    def getJobsList(self):
```

Comment:
This is not very pythonic. Please use:
```suggestion
    def get_jobs_list(self):
```
instead. Same for other methods and variables. Python only uses CamelCase in class names.

Personally, I would even remove the "list" part in the name, because if we are getting "jobs" (plural), I think it is clear that it is a list of them.